tutorials/setup-elastic-metal-proxmox-cluster-with-private-networks/index.mdx

```diff
@@ -0,0 +1,103 @@
+---
+meta:
+  title: How to setup a Proxmox cluster on Elastic Metal servers using the Private Networks feature
+  description: This page explains how to setup a Proxmox cluster on Elastic Metal servers using the Private Networks feature
+content:
+  h1: How to setup a Proxmox cluster on Elastic Metal servers using the Private Networks feature
+  paragraph: This page explains how to setup a Proxmox cluster on Elastic Metal servers using the Private Networks feature
+categories:
+  - dedicated servers
+  - elastic-metal
+tags: Proxmox Elastic-Metal KVM-over-IP virtual-machine container
+dates:
+  validation: 2021-10-01
+  posted: 2020-01-22
+---
+
+## Proxmox - Overview
+
+[Proxmox Virtual Environment](https://www.proxmox.com/en/proxmox-ve) - or Proxmox VE is a server virtualization platform, based on Debian Linux. It allows the deployment of management of [KVM-based](https://www.linux-kvm.org/page/Main_Page) virtual machines or [LXC](https://linuxcontainers.org/) containers.
+
+The tool provides a REST-API as well as a graphical web interface to manage the virtual machines and is licensed under the [GNU Affero General Public License, version 3](https://www.gnu.org/licenses/agpl-3.0.html).
+
+<Message type="requirement">
+
+- You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
+- You have at least two [Elastic Metal servers](https://www.scaleway.com/en/dedibox/) with [KVM-over-IP](/dedibox/kvm-over-ip/concepts/#kvm-over-ip) access
+- You have created a [Private Network](ADD_LINK) and linked your Elastic Metal servers to it
```

Comment:
@RoRoJ t'as un lien à nous partager pour le mettre ici stp ?

Reply:
https://www.scaleway.com/en/docs/compute/elastic-metal/how-to/use-private-networks/